enhancements/samples/removed-bootstrapping.md

```diff
@@ -0,0 +1,203 @@
+
+Those scenarios currently include
+- disconnect / restricted network installs
+- IPv6 single stack installs
+
+## Motivation
+
+With the mandate in 4.2 that the  samples operator should mark itself `Degraded` if any imagestream imports
+are still failing after 2 hours from the start of the install, cluster administrators have been caught off 
+guard when the samples operator eventually marks itself `Degraded` in such cases where samples installation 
+is doomed to fail without manual intervention from the cluster administrator.
+
+In particular, the options available are:
+- mirror desired imagestream images (where typically there are multiple versions per imagestream) to 
+an available and functional registry; and mirror them in such a way such that the repository and image spec
+match what is used on registry.redhat.io
+- since there are many imagestreams, most likely add the imagestreams they are not concerned with to the 
+skip list in the operator config object
+- override the registry used by the samples operator in its config object
+- or if you want to punt on samples entirely, mark the operator as `Removed`
+
+### Goals
+
+1) Alleviate out of the install concerns in scenarios where samples installation will fail out of the gate
+
+2) Allow the cluster administrator time to decide what to do with samples in the cluster
+
+### Non-Goals
+
+Other usability improvements around installing samples from registries other than registry.redhat.io have 
+been identified in Jira:
+
+- regular expressions for further manipulation of the image reference in the imagestream
+- modes where failed imagestreams are automatically added to the skip list (where the administrator then
+automatically can have them added to the management set by fixing the import by mirroring the images, fixing
+the registry in question, etc.)
+- moving the samples operator out of the payload / CVO and making it an OLM operator, where the cluster
+administrator controls which imagestreams are installed when the provision the OLM operator
+
+Bootstrapping as removed is considered the meets minimum, least impacting change to the samples operator 
+that addresses the scenarios where out of the box imagestreams pointing to registry.redhat.io will fail.
+
+## Proposal
+
+Out startup today, the samples operator bootstraps as `Managed`.  That boostrapping can be made conditional,
+and instead come up as `Removed` when the samples operator sees
+
+- Via the techniques noted in the "OCP 4.3 single stack IPv6 control stack" document at
+https://docs.google.com/document/d/1zznvkK87y7UvOJZqs2DM4oWzx94P8Lg6YUb43whil3g/edit#heading=h.q0ahufbae2kz the 
+samples operator determines that it is in an IPv6 single stack scenario
+- By inspecting the image of its own pod, if the samples operator sees something other than 
+`registry.svc.ci.openshift.org` or `registry.redhat.io` assume it is a disconnected / restricted network install
```

Comment:
Is there any acceptable scenario where there is a _non_ transparent proxy here?

Reply:
Only if a non transparent proxy can show up in the image reference for a pod backing a CVO operator ....  I would presume that means it shows up at the install payload level, but that is an educated guess on my part.

@ashcrow @bparees do you know if that is the case ?  Or who should we pull in to confirm one way or the other?

Reply:
see https://github.com/openshift/enhancements/pull/172#discussion_r366535986